tests/queries/0_stateless/02010_lc_native.python

```diff
@@ -8,7 +8,6 @@ import uuid
 CLICKHOUSE_HOST = os.environ.get('CLICKHOUSE_HOST', '127.0.0.1')
 CLICKHOUSE_PORT = int(os.environ.get('CLICKHOUSE_PORT_TCP', '900000'))
 CLICKHOUSE_DATABASE = os.environ.get('CLICKHOUSE_DATABASE', 'default')
-CLICKHOUSE_QUERY_ID = uuid.uuid4().hex
```

Comment:
Nice catch. By the way we can simply send empty string instead of query_id and server will take care about this (create random UUID for query_id by itself).